test-utils/testlib/src/node_user.rs

```diff
@@ -104,4 +121,15 @@ impl NodeUser for ThreadNodeUser {
     fn get_best_block_index(&self) -> u64 {
         self.client.beacon_client.chain.best_index()
     }
+    fn get_shard_blocks_by_index(
+        &self,
+        r: GetBlocksByIndexRequest,
+    ) -> Result<SignedShardBlocksResponse, String> {
+        let start = r.start.unwrap_or_else(|| self.client.shard_client.chain.best_index());
+        let limit = r.limit.unwrap_or(25);
```

Comment:
25 seems arbitrary

Reply:
Same constant as in the RPC API. There is not good place where we could have this constant to be declared in one place though.